### DISCUSSION TITLE
addNodes do not work when apply-default = false

### DISCUSSION THREAD

**Discussion by carrey-feng** [bcakmakoglu/vue-flow#1714] [Q&A]
Why addNodes 、addEdges  do not work when apply-default = false

**Comment by bcakmakoglu** [comment]
You have to handle changes yourself if you disable auto change application.
[Read this section of the docs.](https://vueflow.dev/guide/controlled-flow.html)

**Reply by carrey-feng** [reply]
> You have to handle changes yourself if you disable auto change application. [Read this section of the docs.](https://vueflow.dev/guide/controlled-flow.html)

Thank you for your response. I thought it might have been due to my approach being incorrect.
